daffodil-core/src/main/scala/org/apache/daffodil/dpath/Expression.scala

```diff
@@ -1629,22 +1629,22 @@ case class FunctionCallExpression(functionQNameString: String, expressions: List
         FNExactlyOneExpr(functionQNameString, functionQName, args)
 
       case (RefQName(_, "leftShift", DFDLX), args) =>
-        DFDLXShiftExpr(functionQNameString, functionQName, args,
+        DFDLXBitBinaryExpr(functionQNameString, functionQName, args,
           DFDLXLeftShift(_, _))
       case (RefQName(_, "rightShift", DFDLX), args) =>
-        DFDLXShiftExpr(functionQNameString, functionQName, args,
+        DFDLXBitBinaryExpr(functionQNameString, functionQName, args,
```

Comment:
left shift and right shift should still use DFDLXShiftExpr. Although they take two arguments like BinaryExpr, they have different restrictions. For example, the second argument must always be a unsigned int for shift. So it's sort of like unary, but with an extra argument. I think keeping these two as DFDLXShiftExpr makes sense. No rename or anything needed here.